masterlist.yaml

```
@@ -6,7 +6,7 @@ prelude:
   # Message Anchors
     - &alreadyInX
       type: error
-      content: 'Delete. Already included in {0}.'
+      content: 'Delete. Already included in {0}.'  
```

```suggestion
      content: 'Delete. Already included in {0}.'
```

These trailing spaces were somehow introduced in the last commit.

---

Sorry 😅 Not even touching that area so im not sure why, should be fixed now